zip-0032.rst

```diff
@@ -0,0 +1,464 @@
+
+For the master extended spending key, *depth* is 0, *parent_fvk_tag* is 4 zero bytes,
+and *i* is 0.
+
+When encoded as Bech32, the Human-Readable Part is ``secret-extended-key-main``
+for the production network, or ``secret-extended-key-test`` for the test network.
+
+Sapling extended full viewing keys
+----------------------------------
+
+A Sapling extended full viewing key (*ak*, *nk*, *ovk*, *dk*, *c*), at depth *depth*,
+with parent full viewing key tag *parent_fvk_tag* and child number *i*, is
+represented as a byte sequence:
+
+  I2LEOSP\ :sub:`8`\ (*depth*) || *parent_fvk_tag* || I2LEOSP\ :sub:`32`\ (*i*) || *c* || EncodeXFVKParts(*ak*, *nk*, *ovk*, *dk*)
+
+For the master extended full viewing key, *depth* is 0, *parent_fvk_tag* is 4 zero bytes,
+and *i* is 0.
+
+When encoded as Bech32, the Human-Readable Part is ``zxviews`` for the production
+network, or ``zxviewtestsapling`` for the test network.
+
+Sprout extended spending keys
+-----------------------------
+
+A Sprout extended spending key (*a*\ :sub:`sk`\ , *c*), at depth *depth*, with
+parent address tag *parent_addr_tag* and child number *i*, is represented as a
+byte sequence:
+
+  I2LEOSP\ :sub:`8`\ (*depth*) || *parent_addr_tag* || I2LEOSP\ :sub:`32`\ (*i*) || *c* || EncodeASK(*a*\ :sub:`sk`\ )
+
+For the master extended spending key, *depth* is 0, *parent_addr_tag* is 4 zero bytes,
+and *i* is 0.
+
+When encoded as Bech32, the Human-Readable Part is ``zxsprout`` for the production
+network, or ``zxtestsprout`` for the test network. Sprout extended spending keys
+are encoded using Bech32 even though other Sprout keys and addresses are encoded
+using Base58Check.
+
+
+Test Vectors
+============
+
+TBC
+
+
+Reference Implementation
+========================
+
+https://github.com/zcash/zcash/pull/XXXX
+
+
+References
+==========
+
+.. [#RFC2119] `Key words for use in RFCs to Indicate Requirement Levels <https://tools.ietf.org/html/rfc2119>`_
+.. [#bip-0032] `BIP 32: Hierarchical Deterministic Wallets <https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki>`_
+.. [#bip-0039] `BIP 39: Mnemonic code for generating deterministic keys <https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki>`_
+.. [#bip-0043] `BIP 43: Purpose Field for Deterministic Wallets <https://github.com/bitcoin/bips/blob/master/bip-0043.mediawiki>`_
+.. [#bip-0044] `BIP 44: Multi-Account Hierarchy for Deterministic Wallets <https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki>`_
+.. [#slip-0044] `SLIP 44: Registered coin types for BIP-0044 <https://github.com/satoshilabs/slips/blob/master/slip-0044.md>`_
+.. [#bip-0173] `BIP 173: Base32 address format for native v0-16 witness outputs <https://github.com/bitcoin/bips/blob/master/bip-0173.mediawiki>`_
+.. [#sapling-spec] `Zcash Protocol Specification, Version 2018.0-beta-21 or later [Overwinter+Sapling] <https://github.com/zcash/zips/blob/master/protocol/sapling.pdf>`_
```

Comment:
update these links to protocol.pdf

Reply:
The filename changes weren't made until 2018.0-beta-25. But it's correct to update it to protocol.pdf if the version is changed to be later than that.